app/consumers/compliance_reports_consumer.rb

```diff
@@ -29,6 +31,14 @@ def send_validation(validation)
 
   private
 
+  def notify_payload_tracker(status, status_msg = '')
+    PayloadTracker.deliver(
+      account: @msg_value['account'], system_id: @msg_value['id'],
+      payload_id: @msg_value['payload_id'], status: status,
```

Comment:
This should be `@msg_value['request_id']` - there is no `payload_id` field and the delivery always fails if the field is missing